tests/unit/ReportUtilsTest.ts

```diff
@@ -1180,4 +1181,190 @@ describe('ReportUtils', () => {
+            const chatReport: Report = {
+                ...LHNTestUtils.getFakeReport(),
+                hasOutstandingChildRequest: true,
+            };
+            const currentReportId = '3';
+            const isInFocusMode = true;
+            const betas = [CONST.BETAS.DEFAULT_ROOMS];
+            expect(
+                ReportUtils.shouldReportBeInOptionList({report: chatReport, currentReportId, isInFocusMode, betas, policies: {}, doesReportHaveViolations: false, excludeEmptyChats: false}),
+            ).toBeTruthy();
+        });
+
+        it('should return true when the report has valid draft comment', async () => {
+            const report = LHNTestUtils.getFakeReport();
+            const currentReportId = '3';
+            const isInFocusMode = false;
+            const betas = [CONST.BETAS.DEFAULT_ROOMS];
+
+            await Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT_DRAFT_COMMENT}${report.reportID}`, 'fake draft');
+
+            expect(
+                ReportUtils.shouldReportBeInOptionList({report, currentReportId, isInFocusMode, betas, policies: {}, doesReportHaveViolations: false, excludeEmptyChats: false}),
+            ).toBeTruthy();
+        });
+
+        it('should return true when the report is pinned', () => {
+            const report: Report = {
+                ...LHNTestUtils.getFakeReport(),
+                isPinned: true,
+            };
+            const currentReportId = '3';
+            const isInFocusMode = false;
+            const betas = [CONST.BETAS.DEFAULT_ROOMS];
+            expect(
+                ReportUtils.shouldReportBeInOptionList({report, currentReportId, isInFocusMode, betas, policies: {}, doesReportHaveViolations: false, excludeEmptyChats: false}),
+            ).toBeTruthy();
+        });
+
+        it('should return true when the report is unread and we are in the focus mode', async () => {
+            const report: Report = {
+                ...LHNTestUtils.getFakeReport(),
+                lastReadTime: '1',
+                lastVisibleActionCreated: '2',
+                type: CONST.REPORT.TYPE.CHAT,
+                participants: {
+                    '1': {
+                        notificationPreference: 'always',
+                    },
+                },
+                lastMessageText: 'fake',
+            };
+            const currentReportId = '3';
+            const isInFocusMode = true;
+            const betas = [CONST.BETAS.DEFAULT_ROOMS];
+
+            await Onyx.merge(ONYXKEYS.SESSION, {
+                accountID: 1,
+            });
+
+            expect(
+                ReportUtils.shouldReportBeInOptionList({report, currentReportId, isInFocusMode, betas, policies: {}, doesReportHaveViolations: false, excludeEmptyChats: false}),
+            ).toBeTruthy();
+        });
+
+        it('should return true when the report is an archived report and we are in the default mode', async () => {
```

Comment:
Can you add the inverse check too? should return false when we are in focus mode?